src/middlewares/errorMiddleware.js

```diff
@@ -0,0 +1,19 @@
+import { ApiError } from '../exeptions/api.error.js';
+import { ConsoleLoger } from '../untils/consoleLoger.js';
+
+export const errorMiddleware = (error, req, res, next) => {
+  ConsoleLoger.error('Error: ', error);
```

Comment:
There is a typo in 'ConsoleLoger'. It should be 'ConsoleLogger'. Ensure that the import statement and all usages are corrected.